dev-libs/criterion/criterion-2.4.0_rc.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
On bumps, please drop 3.7 as it's a noop (eclass ignores it).

Reply:
Will do, thanks @thesamesam